frontend-student/src/components/signup/Signup.jsx

```
@@ -27,149 +47,345 @@ export default function Signup() {
+              </button>
+            </div>
+          </form>
+
+          <div className='mt-8 text-center'>
+            <p className='text-gray-600'>
+              Already have an account?{' '}
+              <button
+                className='text-blue-600 font-medium hover:text-blue-800 transition-colors'
+                onClick={() => navigate('/login')}
+              >
+                Log In
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)